docs/tutorials/writing_tutorials.md

```diff
@@ -0,0 +1,315 @@
+---
+title: "Writing executable tutorials"
+description: "How to write an executable tutorial"
+draft: false
+images: []
+menu:
+  docs:
+    parent: "tutorials"
+weight: 110
+toc: true
+---
+
+### The terminology: Stages, chunks and commands
+
+Executable tutorials work a bit like a CI workflow. The command is the smallest
+element of them. It corresponds to an executable with its arguments to be called
+onto the system. A chunk is a collection of commands and is used to group them
+with an environment of execution, chunks can be executed in parallel of each
+other within the same stage. And finally, there are the stages, which are a
+collection of chunks. Stages gets executed one after the other.
+
+#### Code chunks
+
+In an executable tutorial, every line gets executed. Chunks that are executed are
+chunks that provide a valid json object with at least a stage configured in it.
+
+The code fend to create an executable chunk needs to have at least 3 back quotes
```

Comment:
code fence in place of fend typo (I think)

Reply:
ah yes! I have to fix that